src/v/cluster/cluster_link/errc.cc

```diff
@@ -64,6 +64,8 @@ std::string errc_category::message(int c) const {
         return "Topic property is excluded from mirroring";
     case errc::scram_configuration_invalid:
         return "SCRAM configuration invalid";
+    case errc::unknown_error:
+        return "Unknown error";
```

Comment:
Why do we need this very generic error ? Can we map the error to a know one ? This doesn't tell anything to the caller